ToDoList-frontend/src/app/buckets/bucket-edit/bucket-edit.component.ts

```diff
@@ -124,9 +124,6 @@ export class BucketEditComponent implements OnInit {
     this.bucketService.putBucket(environment.bucketEndpoint + this.currentBucketId, this.currentBucket).subscribe(
```

Comment:
metoda `putBucket` powinna przyjąć jedynie `this.currentBucketId` i `this.currentBucket`. Parametr `environment.bucketEndpoint` nie jest potrzebny. Ta metoda powinna mieć go zahardkodowanego jako string wewnatrz siebie. Nie ma potrzeby umieszczac endpointow w environment. Niezależnie od tego, na jakim environment bedziemy: dev, qa, prod, ścieżka względna w endpoincie zawsze będzie taka sama. Jedynie domena się będzie zmieniać, a domenę powinien nam automatycznie doklejać api interceptor.